src/Components/Todo/Todo.js

```diff
@@ -1,13 +1,51 @@
-import React, { memo } from 'react';
+import React, { memo, useState, useRef, useEffect } from 'react';
 
-const todo = props => (
-  <li>
-    <div className="view">
-      <input className="toggle"/>
-      <label>dljf</label>
-      <button className="destroy"/>
-    </div>
-  </li>
-)
+const Todo = props => {
+  const clickInput = useRef(null);
+  const { todo, getTodoEditingId, todoEditingId, onEditedTodo, index, markCompletedTodo, removeTodo } = props;
+  const [text, setText] = useState(todo.text);
+  const isEditing = todoEditingId === todo.id;
+  useEffect(() => {
+    if (isEditing) {
+      clickInput.current.focus();
+    }
+  })
```

Comment:
should add `[your dependencies]` in `useEffect` to prevent re-read the `useEffect`